test/functional/JIT_Test/testng.xml

```diff
@@ -46,6 +46,7 @@
       <class name="jit.test.jitt.array.intArrayLoadStore"/>
       <class name="jit.test.jitt.array.intArrayLoadStoreConstIndices"/>
       <class name="jit.test.jitt.array.shortArrayLoadStore"/>
+      <class name="jit.test.jitt.array.multianewarray"/>
```

Comment:
btw, adding the class name under `arrayTest ` means any test target in [playlist.xml](https://github.com/eclipse/openj9/blob/master/test/functional/JIT_Test/playlist.xml) uses `-testnames arrayTest ` will run this test. That is, 
jit_jitt
jit_jitt_openj9_none_SCC
jit_jitt_none_SCC
jit_jitt_XCEEHDLR

Please ignore this comment if the change is intended. I just want to point it out so we do not run this test in the wrong test target or miss any cases.